genIcons.js

```diff
@@ -0,0 +1,15 @@
+const { execSync } = require('child_process')
+const camelCase = require('lodash.camelcase')
+
+const icon = process.argv[2] || ''
+// converts svg icon name to generated js file name (icon-name.svg->IconName.js)
+function getGenerateIconName() {
+  const pascalCaseName = camelCase(icon)[0].toUpperCase() + icon.slice(1)
+  const [convertedIconName] = pascalCaseName.split('.')
+  return `${convertedIconName}.js`
+}
+execSync(
+  // eslint-disable-next-line max-len
+  `npx @svgr/cli --icon -d renderer/components/Icon ${icon} && npm run lint-fix -- renderer/components/Icon/${getGenerateIconName()}`,
```

Comment:
This should use `lint-fix-base` rather than `lint-fix` as the former allows you to pass your lint paths to it whilst the later applies a default lint path of `"./**/*.{js,json,md}"`